### PR TITLE
Add an example to display OpenStreetMap-sourced data on the map view

### PR DESCRIPTION
**Pull request by abey79**
### What

That serves as a basic example on how to source OpenStreetMap data and display it on a map view. Included both online and in-viewer.

image

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/8044?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/8044?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!
* [x] If have noted any breaking changes to the log API in `CHANGELOG.md` and the migration guide

- [PR Build Summary](https://build.rerun.io/pr/8044)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

File: examples/python/osm_data/osm_data.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+import rerun as rr
+import rerun.blueprint as rrb
+
+CACHE_DIR = Path(__file__).parent / "cache"
+if not CACHE_DIR.exists():
+    CACHE_DIR.mkdir()
+
+OVERPASS_API_URL = "https://overpass-api.de/api/interpreter"
+
+# Find some hotels in the area of Lausanne, Switzerland
```
Comment (zehiko) [inline examples/python/osm_data/osm_data.py:20]: such a "randomly" picked location :)

File: examples/python/osm_data/osm_data.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+# This uses the Overpass API query language: https://wiki.openstreetmap.org/wiki/Overpass_API/Overpass_QL
+QUERY = """
```
Comment (zehiko) [inline examples/python/osm_data/osm_data.py:22]: ultra super nit: ``HOTELS_IN_THE_AREA_QUERY`` (as I wondered below before I read the comment here)

**Review by zehiko — APPROVED** [review]

File: examples/python/osm_data/osm_data.py (hunk 1 above)
Comment (abey79) [reply examples/python/osm_data/osm_data.py:20]: 😇

File: examples/python/osm_data/osm_data.py (hunk 2 above)
Comment (abey79) [reply examples/python/osm_data/osm_data.py:22]: good call, fixed

**Comment by abey79** [comment]
@rerun-bot full-check

**Pull request merged**